.github/workflows/release-extension.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           echo export const gitCommitSha = \"$GIT_COMMIT_SHA\" > $VERSION_FILE
           echo export const version = \"$VERSION\" >> $VERSION_FILE
           sed -i '/"version"/c\\t"version": "'$VERSION'",' $MANIFEST_FILE
+          zip /tmp/source.zip -r .
+          mv /tmp/source.zip .
```

Comment:
Why not zip directly to final location?

Reply:
Changed it to -x exclude since i needed to exclude .git anyway.